doc/use_cases.md

```diff
@@ -0,0 +1,49 @@
+
+The bigger set will be filled with models from https://github.com/stan-dev/example-models 
+and models used, e.g. in https://arxiv.org/abs/1905.11916
+
+## Explorative analysis of algorithms
+
+The posteriors have keywords describing them. This can be used to find in which cases some algorithm fails or is significantly more efficient than other. These keywords can be like "unimodal", "multimodal", "funnel", "low_correlation", "high_correlation". In addition, the dimensionality of parameter space is known. This makes it possible to do explorative analysis, first finding in which cases algorithm fails and then comparing to the reference posterior in which way it is failing, or first finding in which cases algorithm efficiency is much different than for others (e.g. low efficiency for "high_correlation" posteriors) and then analysing the reasons.
+
+## Testing implementations of inference algorithms with asymptotic bias
+
+Sometimes it is useful to use algorithms which have asymptotically non-negligible bias. In this use case we can examine in which cases the inference error is acceptable. It is also possible that the error can be quite big for some posterior quantities, but for example if the focus is in the predictions we could focus on looking at the inference error only for predictions.
+
+## Developing new algorithms for interesting models
+
+We aim to include in posteriordb also challenging models and posteriors for which we don't know yet efficient algorithms, e.g. posteriors for which we know the current dynamic HMC in Stan is inefficient. Some of the posteriors might be such that dynamic HMC needs to be run very very long time, and some might be such that even if dynamic HMC would be best we know we assume it's still getting stuck in local modes. We welcome interesting challenging examples even without a reference posterior or if the reference posterior has been computed with model specific algorithm. Having just models in the database, makes it faster for others to experiment with challenging models, too.
+
+## Code examples
+
+Whenever there is some code in internet, it is possible that someone copies that. We aim to have best practices Stan code and priors, so that if someone copies one of our examples we don't spread bad examples. We may also sometimes include models with bad priors (e.g. some models copied from BUGS examples) as they can provide good test posteriors (e.g. weak identifiability) for convergence diagnostics, but such bad priors should be accompanied with a comment stating that they are bad priors.
+
```

Comment:
We should also add that a usecase is to have data and code available for teaching. Here we could add models and data for the ARM book, BDA3 and so forth. Then it would be easy for teachers to just point to posteriordb to give students data and code.

In the long run, hopefully, people will add other books code and data.

Reply:
Can you add that?